src/com/twu/biblioteca/MainMenu.java

```diff
@@ -105,7 +106,7 @@ public int countCheckedOutBooks() {
 
     public void manageBookList() {
         int count = countCheckedOutBooks();
-      if ((this.books.size() - count) == 0) {
+        if (this.books.size() == count) {
```

Comment:
:+1: 

You could _also_ consider getting rid of the `count` variable and calling `countCheckedOutBooks` instead, as the method name is clearer than the variable itself.

Reply:
Ok, thanks.